articles/quickstart/spa/vuejs-beta/_includes/_centralized_login.md

```diff
@@ -0,0 +1,201 @@
+<!-- markdownlint-disable MD041 MD002 -->
+
+## Install the SDK
+
+You can install the [Auth0 Vue SDK](https://github.com/auth0/auth0-vue) using npm:
+
+```bash
+npm install @auth0/auth0-vue@beta
+```
+
+### Register the plugin
+
+To use the SDK in your Vue application, register the plugin with your Vue application by passing the return value of `createAuth0` to `app.use()`.
+
+```js
+import { createAuth0 } from '@auth0/auth0-vue';
+
+const app = createApp(App);
+
+app.use(
+  createAuth0({
+    domain: '<AUTH0_DOMAIN>',
+    client_id: '<AUTH0_CLIENT_ID>',
+    redirect_uri: '<MY_CALLBACK_URL>'
+  })
+);
+
+app.mount('#app');
+```
+
+Adding the plugin will register our SDK using both `provide` and `app.config.globalProperties`, allowing the SDK to be used with both the [Composition API](https://v3.vuejs.org/guide/composition-api-introduction.html) and Options API.
```

Comment:
Consider:

```suggestion
The plugin will register the SDK using both `provide` and `app.config.globalProperties`, allowing the SDK to be used with both the [Composition API](https://v3.vuejs.org/guide/composition-api-introduction.html) and Options API.
```

Also if we're linking the composition API, we should also link to a reference for the options API. This seems like a good one: https://vuejs.org/guide/introduction.html#options-api